src/lib/logger.ts

```diff
@@ -0,0 +1,55 @@
+// eslint-disable no-console
+import { Logger, LoggerFunction, LogLevel } from '@shapeshiftoss/logger'
+
+import { getConfig } from '../config'
+
+const logLevelCSS: Record<Exclude<LogLevel, LogLevel.NONE>, string> = {
```

Comment:
These colors are a little jarring for those who use dark mode:

<img width="723" alt="Screen Shot 2022-05-04 at 4 42 37 pm" src="https://user-images.githubusercontent.com/97164662/166636510-3d1f7d14-57e7-4854-8ca6-df0c6276e28f.png">

Reply:
i think we could get @reallybeard to give us a nice console palette in a further PR

Reply:
Yah how much control do we have over the colors here?

Reply:
@reallybeard it's just passing that CSS to the logger. @cjthompson has just used the default 1995 hot dog stand colors. we can pass whatever CSS we want

Reply:
@0xdef1cafe so you're saying we use comic sans... perfect. I'll look at some console colors

Reply:
![Screen Shot 2022-05-04 at 10 42 57 AM](https://user-images.githubusercontent.com/89934888/166719052-16e18602-31fd-43be-83b4-66df309b365b.png)

```
error.title {
    background: rgba(155, 44, 44, 0.5);
    color: #F56565;
}
error.body {
    background: rgba(99, 23, 27, 0.5);
    color: #fff;
}


warn.title {
    background: rgba(151, 90, 22, 0.5);
    color: #ECC94B;
}

warn.body {
    background: rgba(95, 55, 14, 0.5); 
    color:#fff;
}

info.title {
    background: rgba(36, 62, 161, 0.5);
    color: #7F99FB;
}

info.body {
    background: rgba(21, 37, 95, 0.5);
    color:#fff;
}


```